common/views/components/BasePage/EventPage.js

```diff
@@ -247,14 +251,22 @@ const EventPage = ({ event }: Props) => {
                         (function() {
                           var iframe = document.getElementById('eventbrite-widget-${event.eventbriteId || ''}');
                           var showWidget = document.getElementById('eventbrite-show-widget-${event.eventbriteId || ''}');
+                          showWidget.classList.add('disabled');
```

Comment:
is `disabled` a class of ours? I don't see it in our repo

Reply:
https://github.com/wellcometrust/wellcomecollection.org/pull/3173/files#diff-db894dd50299dc4d3525dfb5690f5369R13